components/core/core.module.ts

```diff
@@ -57,9 +58,11 @@ import {NgModule} from '@angular/core';
     HsSearchModule,
     HsUtilsModule,
     HsToolbarModule,
+    HsGeolocationModule,
   ],
   exports: [],
   providers: [
+    HsGeolocationService,
```

Comment:
You don't need to include provider in core, but in HsGeolocationModule and it will become available everywhere through that.

Reply:
Ok, removed this provider.